src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -609,23 +610,48 @@ private static string GetCommandId(Dictionary<string, long> dic)
         }
 
         internal CleanCompareExchangeTombstonesCommand GetCompareExchangeTombstonesToCleanup(string databaseName, MergedDatabaseObservationState mergedState, ClusterOperationContext context, out CompareExchangeTombstonesCleanupState cleanupState)
+        {
+            return GetCompareExchangeTombstonesToCleanup(databaseName, mergedState, context, diagnosticLogBuilder: null, out cleanupState);
+        }
+
+        internal CleanCompareExchangeTombstonesCommand GetCompareExchangeTombstonesToCleanup(
+            string databaseName,
+            MergedDatabaseObservationState mergedState,
+            ClusterOperationContext context,
+            [CanBeNull] StringBuilder diagnosticLogBuilder,
```

Comment:
`diagnosticLogBuilder` is only used for tests.
I would prefer to avoid adding such code to the production area, but if you do, it may be more readable to call it with a name that explicitly shows this is for testing purposes.
You can also consider adding it as informational logs, and then it would make sense to put it in the production code area.
This comment is just a thought rather than a request.

Reply:
I do not like that, we use TestingStuff for this kind of 'stuff'